media/css/stories/contentful/landing/_featured-story.scss

```diff
@@ -0,0 +1,155 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@use '../../includes/lib' as stories;
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+.c-stories-featured {
+    .mzp-l-content {
+        padding-top: 0;
+    }
+
+    // allow full-width image on mobile
+    .mzp-l-content:first-of-type {
+        padding-bottom: 0;
+
+        @media (max-width: $screen-sm) {
+            padding-left: 0;
+            padding-right: 0;
+        }
+    }
+
+    &-image-link {
+        display: block;
+        aspect-ratio: 3 / 2;
```

Comment:
Question: IE 11 doesn't support aspect ratio, do we need to add a supports tag? It doesn't look too different on browser stack

Reply:
Good call out - I figured this one was okay to gracefully degrade, as it doesn't have a big layout impact